src/app/researcher/study/request/[memberIdentifier]/page.tsx

```diff
@@ -26,7 +26,7 @@ export default async function MemberHome({ params }: { params: { memberIdentifie
 
                 <List>
                     {/* <ListItem>Submit proposal for review</ListItem> */}
```

Comment:
this isn't something you added, but we could remove this commented out ListItem as well